inkscape/svg2shenzhen/export.py

```diff
@@ -457,6 +458,13 @@ def processExportLayer(self):
                     processes.append(p)
                 for p in processes:
                     p.wait()
+
+                    # Open generated file to replace the wrong layer string
+                    mod = open(layer_dest_kicad_path, "r").read()
+                    mod = mod.replace('F.Cu', layer_label)
+                    open(layer_dest_kicad_path, "w").write(mod)
+
+
```

Comment:
Can we not fix this in bitmap2component instead?

Reply:
I'm not familiar with bitmap2component, but I can try to look into that. Are there any build instructions / pointers that can help here?